cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -1140,6 +1148,11 @@
                     .add(TopicReplicaDistributionGoal.class.getName()).toString(),
                 ConfigDef.Importance.HIGH,
                 GOALS_DOC)
+        .define(INTRA_BROKER_GOALS_CONFIG,
+                ConfigDef.Type.LIST,
+                "",
```

Comment:
Shouldn't this be an empty list rather than an empty string?

Reply:
I think both are ok, the return result from `Config.getList()` will be empty list. I will change to empty list since it adheres to this config's type.